### ISSUE TITLE
Document what travel fund can cover

### ISSUE THREAD

**Issue by MylesBorins** [nodejs/admin#337]
I didn't notice any explicit policy in the travel fund for what we will reimburse... is this something we want to be more explicit about?

Examples include:

* travel
* lodging
* food pre diem
* visa fees

Anything I missed? Anything above people don't agree with?

**Comment by mcollina** [comment]
I'm ok. I would actually limit the amount of alchool reimbursable to 1 glass of wine or a pint of beer per day. Or even zero.

**Comment by ChALkeR** [comment]
Limiting alcohol to anything else over zero and mentioning it would encourage alcohol intake, I suppose?

**Comment by ChALkeR** [comment]
@MylesBorins "visa fees" -> "visa fees and related expenses"?

There are some things like legal document translations that could be required for visa applications, but those are not strictly "visa fees" (I think)?

Otherwise, looks good.

**Comment by tniessen** [comment]
I didn't know that the travel fund covers food. @mcollina I like the idea! I am not sure how easy it would be to tell that apart based on receipts, though?

**Comment by mcollina** [comment]
Itemized receipts for food, up to XX$/day, no alchool?

**Comment by MylesBorins** [comment]
I think $50 per day per diem no alcohol seems reasonable thoughts?

**Comment by mcollina** [comment]
+1 with that.

**Comment by maddhruv** [comment]
are we also reimbursing for JSConf tickets

**Comment by maddhruv** [comment]
or are there some reserved tickets/budget for us

**Comment by Trott** [comment]
We (well, Myles) should likely check with the board as to what is allowable legally. For example, I know that at my place of work, state law prohibits me from expensing alcohol.

The fact that we've covered food for people is news to me. I'm not necessarily opposed, but I've never thought to put in for it. I figured it was primarily transportation and lodging.

Unless there is a ton of travel fund action going on that I've somehow missed, we are not covering JSConf EU tickets. That is a separate event.
